src/pykeen/evaluation/evaluator.py

```diff
@@ -76,6 +76,15 @@ def __init__(
         slice_size: Optional[int] = None,
         automatic_memory_optimization: bool = True,
     ):
+        """Initialize the evaluator.
+
+        :param filtered: Should filtered evaluation be performed?
+        :param requires_positive_mask: Does the evaluator need access to the masks?
+        :param batch_size: >0. Evaluation batch size.
+        :param slice_size: >0. The divisor for the scoring function when using slicing
+        :param automatic_memory_optimization: Whether to automatically optimize the sub-batch size during
+            batch size during evaluation with regards to the hardware at hand.
```

Comment:
Typo "optimize the sub-batch size during batch size "

Reply:
6e52f4e3